README.md

```diff
@@ -96,6 +96,24 @@ Format the Cairo source code (using Scarb):
 $ make fmt
 ```
 
+### Step 4: Setup Language Server
```

Comment:
nit: this should be right above `## How to use this template`

Reply:
Thanks! Will move it up